docs/src/modules/components/AppFrame.js

```diff
@@ -205,9 +210,10 @@ function AppFrame(props) {
               data-ga-event-action="language"
             >
               <LanguageIcon />
-            </IconButton>
+              <span className={classes.language}>{t('language')}</span>
```

Comment:
What do you think of keeping "Language" hard-coded in English?  It could make it easier to find its way back to the correct language.